package.json

```diff
@@ -34,11 +34,13 @@
     "mocha-lcov-reporter": "^1.2.0",
     "mock-fs": "^3.11.0",
     "nock": "^8.0.0",
-    "proxyquire": "^1.7.4"
+    "proxyquire": "^1.7.4",
+    "shimmer": "^1.1.0"
   },
   "dependencies": {
     "google-auth-library": "git://github.com/google/google-auth-library-nodejs.git#02f587472ecdc2582f8d5fc754996aa6ef9388a0",
     "moment": "^2.15.0",
-    "request": "^2.75.0"
+    "request": "^2.75.0",
+    "lodash.isfunction": "^3.0.8"
```

Comment:
I suspect `typeof` would be sufficient here and might avoid a new dependency. WDYT?